lua/catppuccin/groups/integrations/feline.lua

```diff
@@ -292,7 +292,12 @@ function M.get()
 	-- workspace loader
 	components.active[2][1] = {
 		provider = function()
-			local Lsp = vim.lsp.util.get_progress_messages()[1]
+			local Lsp
+			if vim.fn.has "neovim-0.10" then
```

Comment:
This is still wrong. It should be `vim.fn.has "nvim-0.10" == 1` as 0 is considered true in lua